WordPressEditor/WordPressEditor/Classes/Processors/EmbedURLProcessor.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+public struct EmbedURLProcessor{
```

Comment:
No need to change this for this PR, but one thing I wanted to bring up is that it'd be nice if we (at some point) changed this to be modularized in a way that we can have instances of `EmbedURLProcessor` for each platform we support (instead of having methods like `isYouTubeEmbed`.

Again, I think the current solution is awesome for the scope of this PR, but I wanted to bring up the point in case we move forward with other PRs.

Reply:
That sounds great – might make sense to even do that once we have a few platforms in there and can get a feel for how it is working with a longer list – the list supported by WP is _very_ long , and this architecture may not scale that well 😅.